src/value_generators/src/DocumentGenerator.cpp

```diff
@@ -153,6 +153,8 @@ using Parser = std::function<O(const Node&, GeneratorArgs)>;
 
 // Pre-declaring all at once
 // Documentation is at the implementations-site.
+const static boost::posix_time::ptime epoch(boost::gregorian::date(1970, 1, 1));
+const static boost::posix_time::ptime max_date(boost::gregorian::date(2030, 1, 1));
```

Comment:
I feel like max_date should be a lot higher. I know it's what we were already using, but it feels quite stark now.

Reply:
2050?

Reply:
2150?